ci-jobs/create-release-branch.yml

```diff
@@ -0,0 +1,31 @@
+pool:
+  vmImage: 'macOS 10.14'
```

Comment:
Why are we using osx for the job? Are we building anything os specific?

Reply:
No that's a good catch. This pipeline doesn't build WDA.

Reply:
I'll use Linux.

Reply:
Make Linus happy :D